app/models/form/lettings/subsections/setup.rb

```diff
@@ -10,14 +10,22 @@ def pages
     @pages ||= [
       organisation_page,
       stock_owner_page,
+      Form::Lettings::Pages::MinRentValueCheck.new(nil, nil, self),
+      Form::Lettings::Pages::MaxRentValueCheck.new(nil, nil, self),
       managing_organisation_page,
       created_by_page,
       Form::Lettings::Pages::NeedsType.new(nil, nil, self),
       Form::Lettings::Pages::Scheme.new(nil, nil, self),
       Form::Lettings::Pages::Location.new(nil, nil, self),
+      Form::Lettings::Pages::MinRentValueCheck.new(nil, nil, self),
+      Form::Lettings::Pages::MaxRentValueCheck.new(nil, nil, self),
       Form::Lettings::Pages::Renewal.new(nil, nil, self),
       Form::Lettings::Pages::TenancyStartDate.new(nil, nil, self),
+      Form::Lettings::Pages::MinRentValueCheck.new(nil, nil, self),
+      Form::Lettings::Pages::MaxRentValueCheck.new(nil, nil, self),
       Form::Lettings::Pages::RentType.new(nil, nil, self),
+      Form::Lettings::Pages::MinRentValueCheck.new(nil, nil, self),
+      Form::Lettings::Pages::MaxRentValueCheck.new(nil, nil, self),
```

Comment:
It seems ugly to have to add the value check so many times to the setup section but all are relevant, have made this field optional to be consistent with how this is handled for sales logs